doc/contributing/maintaining-cjs-module-lexer.md

```diff
@@ -0,0 +1,54 @@
+# Maintaining cjs-module-lexer
+
+The [cjs-module-lexer](https://github.com/nodejs/node/tree/master/deps/cjs-module-lexer)
+dependency is used within the Node.js ESM implementation to detect the
+named exports of a CommonJS module.
+
+It is used within
+[.../lib/internal/modules/translators](https://github.com/nodejs/node/blob/master/lib/internal/modules/esm/translators.js)
```

Comment:
```suggestion
[lib/internal/modules/esm/translators](https://github.com/nodejs/node/blob/master/lib/internal/modules/esm/translators.js)
```
I don't think it's necessary to include a relative path to the aforementioned file.